app/marketing/views.py

```diff
@@ -269,6 +269,30 @@ def email_settings(request, key):
     email = ''
     level = ''
     msg = ''
+    email_types = dict([('welcome_mail', 'Welcome Email'),('roundup', 'Round Up'), ('new_bounty_notifications', 'New Bounty Notifications Emails'), ('important_product_updates', 'Product Updates Emails'), ('tip', 'Tip Emails'), ('faucet', 'Faucet Notification Emails'), ('bounty', 'Bounty Notification Emails'), ('bounty_match', 'Bounty Match Emails'), ('bounty_feedback', 'Bounty Feedback Emails'), ('bounty_expiration', 'Bounty Expiration Warning Emails'), ('featured_funded_bounty', 'Featured Funded Bounty Emails')])
```

Comment:
Why are you creating this list of tuples and then casting it to a `dict` instead of just writing out the dict directly?